File: application/models/Entities/DataHistory.php
<?php

namespace Entities;

use Doctrine\ORM\Mapping as ORM;

/**
 * Entities\DataHistory
 */
class DataHistory
{
    /**
     * @var integer $id
     */
    private $id;

    /**
     * @var datetime $date
     */
    private $date;

    /**
     * @var string $ip
     */
    private $ip;

    /**
     * @var string $country
     */
    private $country;

    /**
     * @var Entities\File
     */
    private $file;


    /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set date
     *
     * @param datetime $date
     * @return DataHistory
     */
    public function setDate($date)
    {
        $this->date = $date;
        return $this;
    }

    /**
     * Get date
     *
     * @return datetime 
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * Set ip
     *
     * @param string $ip
     * @return DataHistory
     */
    public function setIp($ip)
    {
        $this->ip = $ip;
        return $this;
    }

    /**
     * Get ip
     *
     * @return string 
     */
    public function getIp()
    {
        return $this->ip;
    }

    /**
     * Set country
     *
     * @param string $country
     * @return DataHistory
     */
    public function setCountry($country)
    {
        $this->country = $country;
        return $this;
    }

    /**
     * Get country
     *
     * @return string 
     */
    public function getCountry()
    {
        return $this->country;
    }

    /**
     * Set file
     *
     * @param Entities\File $file
     * @return DataHistory
     */
    public function setFile(\Entities\File $file = null)
    {
        $this->file = $file;
        return $this;
    }

    /**
     * Get file
     *
     * @return Entities\File 
     */
    public function getFile()
    {
        return $this->file;
    }
}

File: AndroidApp/CubbyHole/src/com/supinfo/cubbyhole/mobileapp/models/DataHistory.java
package com.supinfo.cubbyhole.mobileapp.models;

import java.util.Date;

/**
 * Created by anthonyvialleton on 03/04/14.
 */

public class DataHistory {

    private int id;
    private Date date;
    private String ip;
    private String country;

    private File file;

    public DataHistory(){}

    public DataHistory(int id, Date date, String ip, String country, File file) {
        this.id = id;
        this.date = date;
        this.ip = ip;
        this.country = country;
        this.file = file;
    }

    public File getFile() {
        return file;
    }

    public void setFile(File file) {
        this.file = file;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public String getIp() {
        return ip;
    }

    public void setIp(String ip) {
        this.ip = ip;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }


}
